dist/rpm/singularity.spec.in

```diff
@@ -36,6 +36,11 @@ License: BSD-3-Clause-LBNL
 URL: https://www.sylabs.io/singularity/
 Source: %{name}-%{version}.tar.gz
 ExclusiveOS: linux
+# RPM_BUILD_ROOT wasn't being set ... for some reason
+%if "%{sles_version}" == "11"
```

Comment:
This could have been incorporated into the %if immediately following

Reply:
@DrDaveD I agree. That's why I thought we should do some refactoring on this soon, but didn't want to hold up our release coming up.